core/src/main/java/org/elasticsearch/transport/TransportInfo.java

```diff
@@ -109,6 +70,32 @@ public void writeTo(StreamOutput out) throws IOException {
         }
     }
 
+    static final class Fields {
+        static final String TRANSPORT = "transport";
+        static final String BOUND_ADDRESS = "bound_address";
+        static final String PUBLISH_ADDRESS = "publish_address";
+        static final String PROFILES = "profiles";
+    }
+
+    @Override
+    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
```

Comment:
nope, but I moved up readFrom and writeTo and made readFrom the new constructor, that is what bothers github I think